contentcuration/contentcuration/models.py

```diff
@@ -1244,6 +1243,13 @@ class AssessmentItem(models.Model):
     deleted = models.BooleanField(default=False)
 
 
+class SlideshowSlide(models.Model):
+    contentnode = models.ForeignKey('ContentNode', related_name="slideshow_slides", blank=True, null=True,
+                                    db_index=True)
+    sort_order = models.IntegerField(default=1)
+    caption = models.TextField(default="")
+
```

Comment:
Had you had a chance to talk with @radinamatic and @jtamiace about distinguishing between a caption, and descriptive text for accessibility purposes?

Reply:
I will definitely be doing that! Thanks Richard